bazel/dependencies/dropbear/0000-allow-blank-password.patch

```diff
@@ -0,0 +1,57 @@
+diff --git runopts.h runopts.h
+index 1675836..a42e99b 100644
+--- runopts.h
++++ runopts.h
+@@ -107,6 +107,7 @@ typedef struct svr_runopts {
+ 	int noauthpass;
+ 	int norootpass;
+ 	int allowblankpass;
++ int forceblankpass;
```

Comment:
Nit: indentation

Reply:
I changed my .vimrc settings recently, and looks like I broke the tab configuration. Thanks for noticing, fixed.